chat-plugins/mafia.js

```diff
@@ -361,9 +361,10 @@ class MafiaTracker extends Rooms.RoomGame {
 		if (!player) return false;
 		if (!(target in this.players) && target !== 'nolynch') return false;
 		if (!this.enableNL && target === 'nolynch') return false;
-		if (player.lynching || (target === player.userid && !this.selfEnabled)) return false;
+		if (target === player.userid && !this.selfEnabled) return false;
 		if (target === player.userid && (this.hammerCount - 1 > (this.lynches[target] ? this.lynches[target].count : 0)) && this.selfEnabled === 'hammer') return false;
-		if (player.lastLynch + 2000 >= Date.now()) return user.sendTo(this.room, `|error|You must wait another ${Chat.toDurationString((player.lastLynch + 2000) - Date.now()) || '0 seconds'} before you can change your lynch.`);
+		if (player.lastLynch + 2000 >= Date.now()) return user.sendTo(this.room, `|error|You must wait another ${Chat.toDurationString((player.lastLynch + 2000) - Date.now()) || '1 second'} before you can change your lynch.`);
+		if (player.lynching) this.unlynch(Users(player.userid), true);
```

Comment:
If we are having auto-ul for lynch change, we should like have it announce that they changed vote from x player to y player.